src/components/tables/Processes.tsx

```diff
@@ -164,11 +164,11 @@ export function ProcessesTable({ initialTableSettings, renderActions }: Processe
     );
 
     const renderSubComponent = useCallback(({ row }: { row: Row<ProcessV2> }) => {
-        const { last_status, last_step, info } = row.values;
```

Comment:
Not sure why this fixes it; the ProcessV2 type still has properties `last_status` and `last_step`

Reply:
I guess the typing is misleading and that it is actually referring to the `id` properties of the columns?

<img width="446" alt="image" src="https://github.com/workfloworchestrator/orchestrator-core-gui/assets/1482100/29435430-9bfb-4c94-952e-784a72f13a5e">

Reply:
Yeah ok, that's it. Well, no point improving the typing in the old gui.